EnglishFilter/sections/specific.txt

```diff
@@ -256,6 +256,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 purposegames.com##.lladbar
+myanimelist.net##div[class$="banner"]
```

Comment:
1. Too wide rule.
2. This is not an ad.
3. Add `myanimelist.net##.header-mini-banner` to the Annoyances -> mobile-app.txt.
4. Add `myanimelist.net##.footer-mini-banner` to the Annoyances -> self-promo.txt.

Reply:
Okay. Yes, it should be in Annoyances.